src/main/java/com/stripe/model/issuing/Authorization.java

```diff
@@ -434,6 +432,10 @@ public Authorization decline(AuthorizationDeclineParams params, RequestOptions o
     @SerializedName("address_zip_check")
     String addressZipCheck;
 
+    /** One of `success`, `failure`, `exempt`, or `none`. */
+    @SerializedName("authentication")
+    String authentication;
```

Comment:
What is this diff?

Reply:
Another new field that @remi-stripe is in the middle of releasing :) Cf. #838. It should go away if you rebase on latest master.